docs/release-notes/4420-get-experiments-consolidation.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Changes**
+
+-  API: the GetProjectExperiments endpoint (/projects/:id/experiments) is removed and replaced with
```

Comment:
Needs syntax highlighting:
-  API: the ``GetProjectExperiments`` endpoint (``/projects/:id/experiments``) is removed and replaced with
   a *project_id* parameter on the ``GetExperiments`` endpoint (``/experiments``).

Reply:
✅